jalien-xrootd.sh

```diff
@@ -0,0 +1,59 @@
+package: JAlien-XRootD
+version: "%(tag_basename)s"
+tag: 4.8.3
+build_requires:
+ - CMake
+ - "OpenSSL:(?!osx)"
+ - "osx-system-openssl:(osx.*)"
+ - ApMon-CPP
+ - libxml2
+ - "GCC-Toolchain:(?!osx)"
+---
+#! /bin/bash -e
+
+PACKED_NAME="xrootd-$GIT_TAG.tar.gz"
+
+# name of the directory once we unzip
+UNPACKED_NAME="xrootd-$GIT_TAG"
+URL="http://xrootd.org/download/v$GIT_TAG/$PACKED_NAME"
+curl -L -O $URL
+tar xfvz $PACKED_NAME
+
+case $ARCHITECTURE in
+   osx*)
+     [ ! "X$OPENSSL_ROOT" = X ] || OPENSSL_ROOT=`brew --prefix openssl`
```

Comment:
Use bash syntax:

```bash
[[ $OPENSSL_ROOT ]] || OPENSSL_ROOT=$(brew --prefix openssl)
```

Reply:
OK, then this should also be changed in the xrootd.sh recipe as I just ripped it from
there.

Reply:
Just keep the work in a separate PR without merging it so that we don't touch production 🙂